cli.js

```diff
@@ -42,6 +52,265 @@ function init(root, projectName) {
   spawn(path.resolve(__dirname, 'init.sh'), [projectName], {stdio:'inherit'});
 }
 
+function link(libraryPath) {
+  libraryPath = path.relative(process.cwd(), path.resolve(process.cwd(), libraryPath));
+  var packagePath = path.resolve(libraryPath, 'package.json');
+  var pkg;
+  try {
+    pkg = require(packagePath);
+  } catch(err) {
+    pkg = { 'react-native': {} };
+  }
+  var name = pkg['react-native'].xcodeproj || path.basename(libraryPath);
+  var targetName = pkg['react-native'].target || 'RCT' + name.replace(/^(?:RCT)?(.*?)(?:IOS)?$/, '$1');
+  var xcodeprojName = targetName + '.xcodeproj';
+  var staticLibraryPath = pkg['react-native'].staticLibrary || 'lib' + targetName + '.a';
+  var pbxprojPath = path.resolve(process.cwd(), libraryPath,
+                                 xcodeprojName, 'project.pbxproj')
```

Comment:
nit: not a big fan of this style of indentation. I know we don't have coding style published anywhere, but I don't think this formatting is common in our codebase. Also it makes harder to make changes and maintain blame.